### DISCUSSION TITLE
Colorpicker as Button

### DISCUSSION THREAD

**Discussion by LuuukDotDev** [GrapesJS/grapesjs#3567] [Q&A]
Hi,
is it possible to display the color picker on button click in grapesjs?
I'd like to open the color picker by clicking on this button.
![Screenshot 2021-06-25 152346](https://user-images.githubusercontent.com/62956729/123431267-5ff58200-d5c9-11eb-8a13-e236de478163.png)
Thanks for help.

**Comment by artf** [comment]
Hi @lucaamrein check this [reply](https://github.com/artf/grapesjs/issues/3194#issuecomment-752651277)
